### PR TITLE
Armstrong Numbers approach

### PR DESCRIPTION
**Pull request by bobahop**
Approach for Armstrong Numbers

### CODE REVIEW

**Comment by bobahop** [comment]
@wolf99: C approach for Armstrong Numbers ready for review.

File: exercises/practice/armstrong-numbers/.approaches/introduction.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        num /= 10;
+    }
+    return candidate == pow_total;
+}
+```
+
+For more information, check the [`log10` with no `pow` approach][approach-log10-no-pow].
+
+[approach-log10-no-pow]: https://exercism.org/tracks/c/exercises/armstrong-numbers/approaches/log10-no-pow
+
```
Comment (wolf99) [inline exercises/practice/armstrong-numbers/.approaches/introduction.md:53]:
```suggestion
```

File: exercises/practice/armstrong-numbers/.approaches/introduction.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        pow_total += pow_temp_total;
+        num /= 10;
+    }
+    return candidate == pow_total;
+}
+```
+
+For more information, check the [`log10` with no `pow` approach][approach-log10-no-pow].
+
+[approach-log10-no-pow]: https://exercism.org/tracks/c/exercises/armstrong-numbers/approaches/log10-no-pow
```
Comment (wolf99) [inline exercises/practice/armstrong-numbers/.approaches/introduction.md:52]: Is the intention here to refer to the same directory, or did you mean to reference the relatee `content.md` directly?

The former seems a bit odd to me, the reader is already here surely?
The latter needs an update to this URL.

Comment (bobahop) [reply exercises/practice/armstrong-numbers/.approaches/introduction.md:52]: The intent is to link to the subdirectory of log10-no-pow in the approaches directory. If you look at the other approaches, they do the same, unless I'm just not seeing something here.

**Review by wolf99 — APPROVED** [review]

**Pull request merged**